pages/common/history.md

```diff
@@ -13,3 +13,7 @@
 - Overwrite history file with history of current `bash` shell (often combined with `history -c` to purge history):
 
 `history -w`
+
+- Delete the history entry at the specified offset:
```

Comment:
Is this an offset from the beginning or the end of the history?

Reply:
If you issue the `history` command, it becomes very clear:

```bash
marco:~/testdir$ history -c
marco:~/testdir$ ls
a  b  c
marco:~/testdir$ ls -l
total 0
-rw-r--r-- 1 marco marco 0 Jan 11 01:46 a
-rw-r--r-- 1 marco marco 0 Jan 11 01:46 b
-rw-r--r-- 1 marco marco 0 Jan 11 01:46 c
marco:~/testdir$ echo Hello
Hello
marco:~/testdir$ history
 1002  ls
 1003  ls -l
 1004  echo Hello
 1005  history
marco:~/testdir$ history -d 1003
marco:~/testdir$ history
 1002  ls
 1003  echo Hello
 1004  history
 1005  history -d 1003
 1006  history
marco:~/testdir$
```

Reply:
Ah, I see :P I think I misinterpreted this. It's fine as-is then I think :-)